docs/src/administration/users.md

```diff
@@ -3,136 +3,182 @@ title: "User administration"
+Say you want to add `user1@example.com` to the project with **Project Admin** permissions:
 
 ```bash
-platform user:add alice@example.com -r admin
+platform user:add user1@example.com -r admin
 ```
 
-After adding `alice@example.com`, Alice will receive an invitation email asking to confirm her details and optionally, register for a Platform.sh account.
+{{< /codetabs >}}
 
-To give Bob different levels of access depending on the environment type, in the current project:
- 
--  **Viewer** role to the **Production environment**
--  **Contributor** role to **all Staging type environments**
--  **Admin** role to **all Development type environments**
+Once you add a user to a project, they receive an email with instructions.
+For SSH access changes to apply after you add a user to a project, you have to redeploy each environment by either clicking **Redeploy** in the console or running `platform redeploy`.
 
-You would run:
+## Manage existing users
 
-```bash
-platform user:role bob@example.com -r production:viewer -r staging:contributor -r development:admin
-```
+### Delete a user from a project
 
-Use `platform list` to get the full list of commands.
+To delete a user from a project, follow these steps:
 
-## User access and integrations
+{{< codetabs >}}
+---
+title=In the console
+file=none
+highlight=false
+---
+1. In the console, select the project where you want to delete a user.
+2. Under **Settings**, click **Access**.
+3. Select the user you want to delete and click **Delete**.
+4. Click **Save**.
 
-If you have setup an [external integration](/integrations/source/_index.md) to GitHub, GitLab, or Bitbucket, this adds an additional layer of access control to the project that you will need to be aware of.
+<--->
+---
+title=Using the CLI
+file=none
+highlight=false
+---
+Use the [Platform.sh CLI (Command Line Interface)](/development/cli/_index.md) to delete existing users.
 
-For example, if you invite a user with **Project Admin** role to a project on Platform.sh, but you haven't invited them to the remote repository on GitHub / GitLab, they might be [unable to clone the project](/administration/web/_index.md#git) locally.
+`platform user:delete`
 
-In this example, using either `platform get` with the CLI:
+  ```bash
+  platform user:delete user1@example.com
+  ```
 
-```bash
-$ platform get <projectID>
+  After you delete a user from a project or an environment type, you must trigger a redeploy to propagate SSH access changes to each environment. You can redeploy by using the CLI command `platform redeploy`.
 
-```
+{{< /codetabs >}}
 
-or the `git clone` command visible from the "Git" dropdown in the management console
+Once you delete a user, they can no longer access the project anymore.
+After you delete a user from a project or an environment type, you must trigger a redeploy to propagate SSH access changes to each environment.
+You can redeploy by clicking **Redeploy** in the console.
```

Comment:
This should include both ways, as with adding a user.

Reply:
```suggestion
You can redeploy by clicking **Redeploy** in the console or  using the CLI command `platform redeploy`.
```